### PR TITLE
Add the Mock API calls for Login and Student API calls

### PR DESCRIPTION
**Pull request by Rohitgour03**
Add the Mock API calls for Login and Student API calls.

### CODE REVIEW

File: src/mocks/handlers.js [hunk 1]
```diff
@@ -1,36 +1,253 @@
...
-    //  handles a POST add profile request
-    rest.post(
-        "../../api/api_endpoint/api/student/addResume/",
+    // handles a Post request to login
+    rest.post("../../api/api_endpoint/api/login/", (req, res, ctx) => {
+        return res(
+            ctx.json({
+                action: "Login",
+                message: "Verified",
+                user_type: ["student"],
```
Comment (gowtham3105) [inline src/mocks/handlers.js:10]:
```suggestion
                user_type: ["student", “admin”],
```

File: src/mocks/handlers.js [hunk 2]
```diff
@@ -1,36 +1,253 @@
...
-                    details: {
-                        id: "190010036",
-                        resume_list: [{
-                            link: "https://storage.googleapis.com/cdc-backend-attachments/resume/190010036%2F8KIOT3PW1JIS718_CSE-V-SEM.pdf",
-                            name: "8KIOT3PW1JIS718_CSE-V-SEM.pdf",
+                    action: "Get Dashboard - Student",
+                    message: "Data Found",
+                    placements: [{
+                        id: "Q54IRZZMC3RP8F6",
+                        company_details_pdf_links: [{
```
Comment (gowtham3105) [inline src/mocks/handlers.js:53]: Add some real pdf links which will open

Comment (gowtham3105) [reply src/mocks/handlers.js:53]: For all pdf and images search some random pdf or image and keep those links

File: src/mocks/handlers.js [hunk 3]
```diff
@@ -1,36 +1,253 @@
...
+                details: {
+                    id: "190010036",
+                    resume_list: [{
+                        link: "https://storage.googleapis.com/cdc-backend-attachments/resume/190010036%2F8KIOT3PW1JIS718_CSE-V-SEM.pdf",
+                        name: "8KIOT3PW1JIS718_CSE-V-SEM.pdf",
+                    }, ],
+                    offers: [{
+                        designation: "Software Developer",
+                        company_name: "Make My Trip",
+                        application_id: "LLW4STE76GEJYOR",
```
Comment (gowtham3105) [inline src/mocks/handlers.js:30]: See that this application I’d is also returned in getDashboard

**Comment by gowtham3105** [comment]
Also create const variables for large responses and define then in the starting. That will make the code looks good.

**Review by gowtham3105 — CHANGES_REQUESTED** [review]

File: src/mocks/handlers.js [hunk 4]
```diff
@@ -1,36 +1,253 @@
 import { rest } from "msw";
 
 export const handlers = [
-    //  handles a POST add profile request
-    rest.post(
-        "../../api/api_endpoint/api/student/addResume/",
+    // handles a Post request to login
+    rest.post("../../api/api_endpoint/api/login/", (req, res, ctx) => {
```
[the remark this answers is not in the document]
Comment (Rohitgour03) [reply src/mocks/handlers.js:5]: Oh, I got it. My mistake.

File: src/mocks/handlers.js (hunk 2 above)
Comment (Rohitgour03) [reply src/mocks/handlers.js:53]: PDF llink added

**Comment by gowtham3105** [comment]
Did u try running these mocks? @Rohitgour03

**Comment by gowtham3105** [comment]
like I'm seeing some naming differences
like `placements` is shown as `placementsDetails`
check it once try running it and make a video of it

**Pull request closed without merging**